'''예외 처리'''


try:
    print("나누기 전용 계산기")
    nums = []
    nums.append(int(input("첫 번째 숫자를 입력하세요 : ")))
    nums.append(int(input("두 번째 숫자를 입력하세요 : ")))
    # nums.append(int(nums[0] / nums[1])) --> 실수로 까먹었다면
    print("{0} / {1} = {2}".format(nums[0],nums[1],nums[2]))
except ValueError:
    print("에러가 발생하였습니다.")
except ZeroDivisionError as err:
    print(err)
except:     # else 와 비슷한 기능
    print("알 수 없는 오류 발생")
